src/routes/user/user.routes.js

```diff
@@ -0,0 +1,45 @@
+const express = require("express");
+const router = express.Router();
+const passport = require("passport");
+const userAccountController = require("../../controllers/user/user.controller");
+const userValidator = require("../../validators/user/user.validator");
```

Comment:
Make sure that the incoming requests fail automatically if the users try to send any redundant data that is not defined in these request validators.